src/common/timer/interaction-timer.js

```diff
@@ -42,7 +43,10 @@ export class InteractionTimer extends Timer {
       subscribeToVisibilityChange((state) => {
         if (state === 'hidden') this.pause()
         // vis change --> visible is treated like a new interaction with the page
-        else this.refresh()
+        else {
+          this.onResume?.()
```

Comment:
Would not need optional chaining if applying above comment on using `noop` functions.